### PR TITLE
ADEN-8685 Floor Adhesion: hide after viewability

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

**Review by nandy-andy — APPROVED** [review]

File: app/modules/ads/setup.js [hunk 1]
```diff
@@ -54,6 +55,7 @@ export const adsSetup = {
     templateService.register(BigFancyAdAbove, getBfaaConfig());
     templateService.register(BigFancyAdBelow, getBfabConfig());
     templateService.register(FloorAdhesion, getOutOfPageConfig());
+    templateService.register(HideOnViewability, {});
```
Comment (Bielik20) [inline app/modules/ads/setup.js]: Second argument of `templateService.register` is optional, why pass empty object?

File: app/modules/ads/slots.js [hunk 2]
```diff
@@ -207,7 +207,7 @@ export const slots = {
           loc: 'footer',
           rv: 1,
         },
-        defaultTemplates: ['floorAdhesion'],
+        defaultTemplates: ['floorAdhesion', 'hideOnViewability'],
         defaultSizes: [[320, 50], [320, 100], [300, 50]],
```
Comment (Bielik20) [inline app/modules/ads/slots.js]: Order.

File: app/modules/ads/setup.js (hunk 1 above)
Comment (dmnsobczak) [reply app/modules/ads/setup.js]: Yep, removed.

File: app/modules/ads/slots.js (hunk 2 above)
Comment (dmnsobczak) [reply app/modules/ads/slots.js]: That's outside the scope of the ticke... blahblah, changed :D

**Review by Bielik20 — APPROVED** [review]

**Pull request merged**